File: LowestUniqueNumber/lun.py
# Lowest Unique Number solution in Python 2 for CodeEval.com by Steven A. Dunn

import sys

for line in open(sys.argv[1], 'r'):
  nums = line.strip('\n').split(' ')
  lowest = 10
  for i in nums:
    if nums.count(i) == 1 and int(i) < lowest:
      lowest = int(i)
  if lowest == 10: print 0
  else: print nums.index(str(lowest)) + 1

File: LowestUniqueNumber/lun.rb
# lowest unique number soln in rb for codeeval by steven a dunn

File.open(ARGV[0], 'r').each_line do |line|
    nums = line.chomp.split(' ')
    lowest = 10
    nums.each do |i|
        if nums.count(i) == 1 and i.to_i < lowest
            lowest = i.to_i
        end
    end
    if lowest == 10
        puts 0
    else
        puts nums.index(lowest.to_s) + 1
    end
end